raiden/utils/filters.py

```diff
@@ -89,3 +97,42 @@ def decode_event(abi: Dict, log: Dict):
     }
     event_abi = topic_to_event_abi[event_id]
     return get_event_data(event_abi, log)
+
+
+class StatelessFilter(LogFilter):
+    _last_block: int = -1
```

Comment:
this is a instance attribute and not a class attribute, it should be inside the constructor

Reply:
agree with Augusto. In the future we could support multiple chains with multiple numbers, thus we could have filters for different chains with different `last_block`

Reply:
Sorry to jump in here, but having a class variable with a default value that’s later shadowed by an instance variable is a pretty common pattern - i.e. not really seeing a problem here